lib/linguist/languages.yml

```diff
@@ -1619,6 +1619,14 @@ GDScript:
   tm_scope: source.gdscript
   ace_mode: text
   language_id: 123
+GEDCOM:
+  type: data
+  ace_mode: text
+  extensions:
+  - ".ged"
+  - ".gedcom"
```

Comment:
That second extension only has 10 files on GitHub and therefore doesn't match our criteria of *hundreds of repositories*.